docs/guides/force-organizations.mdx

```diff
@@ -86,7 +86,7 @@ This guide will be written for Next.js applications using App Router, but the sa
       export default function Layout() {
         const { orgId } = auth();
 
-        const hasActiveOrg = orgId !== null;
+        const hasActiveOrg = orgId !== undefined;
```

Comment:
we need to check against both, null means that the user is signedOut, undefined means that the user is signed in but the don't have an active org selected.

https://github.com/clerk/javascript/blob/80e64773135865434cf0e6c220e287397aa07937/packages/backend/src/tokens/authObjects.ts#L50
```suggestion
        const hasActiveOrg = !!orgId
```